pages/sentinel_create_samples.vue

```diff
@@ -95,17 +95,20 @@ export default {
       const errored = resp.filter((obj) => Object.keys(obj).includes('errors'))
       if (errored.length > 0) {
         const msg = errored.map((e) => e.errors.join(', ')).join(', ')
-        this.alertMessage = msg
-        this.showDismissibleAlert = true
+        this.showAlert(msg, 'danger')
```

Comment:
Maybe here doing a check if sucessful.length > 0 then showAlert with 'warning' type and a better error message

Reply:
Might also be a cleaner way to handle all this logic. 
- errors no success
- errors and success
- success no errors